### ISSUE TITLE
Browse docs: use local reference if available

### ISSUE THREAD

**Issue by NathanLovato** [GDQuest/emacs-gdscript-mode#63]
We have a feature to browse the online code reference, which is great. But depending on your internet connection it can take a few seconds every time you look up a symbol.

I'd like to provide an option to use an offline build of the docs as anyone can build or download the docs as static HTML files. That would allow opening the code reference for a class instantly.

**Comment by NathanLovato** [comment]
Here's a build of the current stable class reference, but with all class files cleaned up. One issue with the official docs is that each file has 10,000+ lines of code for the menu alone, slowing down parsing and loading. These files open fast in eww.

[classes-cleaned-up.zip](https://github.com/GDQuest/emacs-gdscript-mode/files/4709018/classes-cleaned-up.zip)

I used sed to trim the files, it's fast enough:

```fish
for i in ./_build/html/classes/*.html
    sed -i '/<nav data-toggle="wy-nav-shift"/,/<\/nav>/d; /<script/,/<\/script/d; /<footer/,/<\/footer/d; /<head/,/<\/head/d' $i
end
```

**Comment by NathanLovato** [comment]
Closed with 4fcfc61c0efd0c0992035a3d52bee9d2f1e5effc
